Automattic/themes
# Assembler: Add support for upcoming section style variations

Note that not every theme style variation has a "Dark 2" style (intentionally). Some themes are more minimal than others.  

## Testing 

1. Activate the theme. 
2. Add three patterns. 
3. Add a different section style to each one.
4. Switch to different style variations in the site editor. 
5. See changes. 

## Visuals 
Blueberry: 
![CleanShot 2024-05-31 at 17 19 12](https://github.com/Automattic/themes/assets/1813435/41f402e9-ad80-4596-a81a-35f55e4b3c2b)

Tangerine: 
![CleanShot 2024-05-31 at 17 19 58](https://github.com/Automattic/themes/assets/1813435/027736e0-6c03-45f4-b791-a1c4d028a702)

Mauve: 
![CleanShot 2024-05-31 at 17 19 47](https://github.com/Automattic/themes/assets/1813435/44fcda39-6848-4f97-a7e2-6dbae799f9bd)

Linen: 
![CleanShot 2024-05-31 at 17 19 36](https://github.com/Automattic/themes/assets/1813435/1dfe7d46-faf8-4d21-911b-44177a7bd318)

Rose: 
![CleanShot 2024-05-31 at 17 19 00](https://github.com/Automattic/themes/assets/1813435/7e432551-efbb-4c8a-8ea8-10e9b6aa9596)

Pearl: 
![CleanShot 2024-05-31 at 17 19 23](https://github.com/Automattic/themes/assets/1813435/bdb6cde7-7cd3-4fca-941e-dfd1cd52816e)

Lilac: 
![CleanShot 2024-05-31 at 17 21 33](https://github.com/Automattic/themes/assets/1813435/6452adc8-1b6d-413c-9a44-fc6a3955c266)
